aviary/models/large_single_aisle_1/large_single_aisle_1_GwGm.dat

```diff
@@ -195,6 +195,8 @@ aircraft:engine:data_file=models/engines/turbofan_23k_1.deck
  ROSCAB=9999.,	! cabin rate of sink during descent (500.) fpm
  RSMX=900.,	! maximum allowable rate of sink during landing approach ft per min (1000)
  FRESF=4998,   ! Reserve Fuel fraction of mission fuel (minus sign for code trigger)
+ FRESF2=0,   ! Reserve fuel fraction of mission fuel (negative sign trigger)
+ FRESF3=1,  ! Use Reserve Fuel
```

Comment:
Why the addition of these two lines?

Reply:
If I don't add these two lines, test_Fortran_to_Aviary fails.